backend/app/views/spree/admin/orders/cart.html.erb

```diff
@@ -11,13 +11,12 @@
 
 <% admin_breadcrumb(t('spree.cart')) %>
 
-
 <div data-hook="admin_order_edit_header">
   <%= render partial: 'spree/shared/error_messages', locals: { target: @order } %>
 </div>
 
-<% if @order.payments.exists? && @order.is_risky? %>
-    <%= render 'spree/admin/orders/risk_analysis', latest_payment: @order.payments.reorder("created_at DESC").first %>
+<% if @order.is_risky? %>
```

Comment:
For context, I checked, and that's backward compatible 👍 

```ruby
def is_risky?
  payments.risky.count > 0
end
```

When there're no payments, that'll return `false` (`0 > 0`)

Reply:
Yes, that was my assessment too 👍